packages/desktop-client/src/components/transactions/MobileTransaction.js

```diff
@@ -99,6 +177,789 @@ function Status({ status }) {
+
+  onClick = (transactionId, name) => {
+    let { dateFormat } = this.props;
+
+    this.props.pushModal('edit-field', {
+      name,
+      onSubmit: (name, value) => {
+        let { transactions } = this.state;
+        let transaction = transactions.find(t => t.id === transactionId);
+        // This is a deficiency of this API, need to fix. It
+        // assumes that it receives a serialized transaction,
+        // but we only have access to the raw transaction
+        this.onEdit(serializeTransaction(transaction, dateFormat), name, value);
+      },
+    });
+  };
+
+  // onSplit = () => {
+  //   this.props.navigation.navigate('CategorySelect', {
+  //     title: 'Select the first category',
+  //     onSelect: categoryId => {
+  //       let transactions = this.state.transactions;
+
+  //       // Split the transaction
+  //       let { data } = splitTransaction(transactions, transactions[0].id);
+  //       data[1].category = categoryId;
+
+  //       this.setState({ transactions: data }, this.focusSplit);
+  //     },
+  //   });
+  // };
+
+  // onAddSplit = () => {
+  //   this.props.navigation.navigate('CategorySelect', {
+  //     title: 'Select a category',
+  //     onSelect: categoryId => {
+  //       let transactions = this.state.transactions;
+
+  //       // Split the transaction
+  //       let { data } = addSplitTransaction(transactions, transactions[0].id);
+  //       // Set the initial category
+  //       data[data.length - 1].category = categoryId;
+
+  //       this.setState({ transactions: data }, this.focusSplit);
+  //     },
+  //   });
+  // };
+
+  // focusSplit = () => {
+  //   if (this.lastChildAmount) {
+  //     this.lastChildAmount.focus();
+  //   }
+  // };
+
+  // onDeleteSplit = transaction => {
+  //   let { transactions } = this.state;
+  //   let { data } = deleteTransaction(transactions, transaction.id);
+  //   this.setState({ transactions: data });
+  // };
+
+  // TODO: `renderActions` was used to render swipe actions on split transaction
+  // rows. It enabled you to swipe, revealing a delete action, and if you swiped
+  // all the way it would perform the delete action. The entire swipe was
+  // animated. Swipe actions are a native paradigm and we'll need a web-friendly
+  // replacement.
+  //
+  // renderActions = (progress, dragX) => {
+  //   const trans = dragX.interpolate({
+  //     inputRange: [-101, -100, -50, 0],
+  //     outputRange: [-6, -5, -5, 20],
+  //   });
+  //   return (
+  //     <RectButton
+  //       onPress={this.close}
+  //       style={{
+  //         flex: 1,
+  //         justifyContent: 'center',
+  //         backgroundColor: colors.r4,
+  //       }}
+  //     >
+  //       <Animated.Text
+  //         style={{
+  //           color: 'white',
+  //           textAlign: 'right',
+  //           transform: [{ translateX: trans }],
+  //         }}
+  //       >
+  //         Delete
+  //       </Animated.Text>
+  //     </RectButton>
+  //   );
+  // };
```

Comment:
I agree it makes sense to punt this for later (⏩ https://github.com/actualbudget/actual/issues/1352). Can you remove the relevant commented code since it can be retrieved from the archive later?

Reply:
Done via https://github.com/actualbudget/actual/pull/1340/commits/827802d4ddc29e4117cd37a4494e35ba74069115